src/js/common/schemaform/definitions/address.js

```diff
@@ -225,3 +225,289 @@ export function uiSchema(label = 'Address', useStreet3 = false, isRequired = nul
     }
   };
 }
+
+
+// PCIU Validations
+
+function validateSpecialCharacter(errors, string) {
+  if (string && !isValidSpecialCharacter(string)) {
+    errors.addError(
+      "Please only use letters, numbers, and the special characters #%&'()+,./:@"
+    );
+  }
+}
+
+function validateZIP(errors, zip) {
+  if (zip && !isValidUSZipCode(zip)) {
+    errors.addError('Please enter a valid 5 or 9 digit ZIP (dashes allowed)');
+  }
+}
+
+// PCIU helpers
+function mergeUniques(first, second) {
```

Comment:
This helper is only helpful if the inputs are expected to change, should we simply generate the list and store it here as well?